<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>wavePicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="stationTree">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string>Double click to hide or show; Right click for sorting options.</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(20, 20, 20);</string>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="title">
            <string>Component</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>241</width>
              <height>29</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="compZbtn">
               <property name="toolTip">
                <string>Shortcut Z Trace: 1</string>
               </property>
               <property name="statusTip">
                <string>Shortcut Z Trace: 1</string>
               </property>
               <property name="text">
                <string>Z</string>
               </property>
               <property name="shortcut">
                <string>1</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="compNbtn">
               <property name="toolTip">
                <string>Shortcut N Trace: 2</string>
               </property>
               <property name="statusTip">
                <string>Shortcut N Trace: 2</string>
               </property>
               <property name="text">
                <string>N</string>
               </property>
               <property name="shortcut">
                <string>2</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="compEbtn">
               <property name="toolTip">
                <string>Shortcut E Trace: 3</string>
               </property>
               <property name="statusTip">
                <string>Shortcut E Trace: 3</string>
               </property>
               <property name="text">
                <string>E</string>
               </property>
               <property name="shortcut">
                <string>3</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QToolBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>390</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="tabSpacing">
         <number>6</number>
        </property>
        <widget class="QWidget" name="pageEvents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>258</width>
           <height>366</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Event Picks</string>
         </attribute>
         <widget class="QGroupBox" name="pickGroup">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>270</y>
            <width>256</width>
            <height>60</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Active Picker</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>22</y>
             <width>241</width>
             <height>29</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pickP">
              <property name="toolTip">
               <string>Shortcut P Pick: F1</string>
              </property>
              <property name="statusTip">
               <string>Shortcut P Pick: F1 - P Pick is unique</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pickP:checked{background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);};</string>
              </property>
              <property name="text">
               <string>P</string>
              </property>
              <property name="shortcut">
               <string>F1</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pickS">
              <property name="toolTip">
               <string>Shortcut S Pick: F2</string>
              </property>
              <property name="statusTip">
               <string>Shortcut S Pick: F2 - S Pick is unique</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pickS:checked{background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);};</string>
              </property>
              <property name="text">
               <string>S</string>
              </property>
              <property name="shortcut">
               <string>F2</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pickAmp">
              <property name="toolTip">
               <string>Shortcut Amp Pick: F3</string>
              </property>
              <property name="statusTip">
               <string>Shortcut Amp Pick: F3</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pickAmp:checked{background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);};</string>
              </property>
              <property name="text">
               <string>Amp</string>
              </property>
              <property name="shortcut">
               <string>F3</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pick1">
              <property name="toolTip">
               <string>Shortcut 1 Pick: F5</string>
              </property>
              <property name="statusTip">
               <string>Shortcut 1 Pick: F5</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pick1:checked{background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);};</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="shortcut">
               <string>F5</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pick2">
              <property name="toolTip">
               <string>Shortcut 2 Pick: F6</string>
              </property>
              <property name="statusTip">
               <string>Shortcut 2 Pick: F6</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pick2:checked{background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);};</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
              <property name="shortcut">
               <string>F6</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QTreeWidget" name="eventTree">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>260</width>
            <height>231</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>0</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Double click to change active event or highlight pick.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>231</width>
            <height>29</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="addEventBtn">
             <property name="text">
              <string>Add Event</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="deleteItemBtn">
             <property name="text">
              <string>Delete Item</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="pageFilter">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>258</width>
           <height>366</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Bandpass Filter</string>
         </attribute>
         <widget class="QLabel" name="labelFmax">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>32</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Upper Frequency</string>
          </property>
         </widget>
         <widget class="QPushButton" name="filterButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Shortcut for filter: f</string>
          </property>
          <property name="statusTip">
           <string>Shortcut for filter: f</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(154, 154, 154);
background-color: rgb(158, 234, 116);
alternate-background-color: rgb(255, 140, 140);</string>
          </property>
          <property name="text">
           <string>Filter On</string>
          </property>
          <property name="shortcut">
           <string>CapsLock</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="labelFmin">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Lower Frequency</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="fminSpin">
          <property name="geometry">
           <rect>
            <x>145</x>
            <y>0</y>
            <width>91</width>
            <height>26</height>
           </rect>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="fmaxSpin">
          <property name="geometry">
           <rect>
            <x>145</x>
            <y>30</y>
            <width>91</width>
            <height>26</height>
           </rect>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string extracomment="Hz"> Hz</string>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
         <widget class="QCheckBox" name="zerophaseCheck">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>130</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Zerophase</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelCorner">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>66</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Corners</string>
          </property>
          <property name="buddy">
           <cstring>cornersSpin</cstring>
          </property>
         </widget>
         <widget class="QSpinBox" name="cornersSpin">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>126</y>
            <width>40</width>
            <height>27</height>
           </rect>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
         <widget class="QSlider" name="fminSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>231</width>
            <height>29</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
         <widget class="QSlider" name="fmaxSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>231</width>
            <height>29</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="GraphicsLayoutWidget" name="qtGraphLayout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAs_JSON"/>
    <addaction name="actionLoad_JSON"/>
    <addaction name="separator"/>
    <addaction name="actionExport_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionExport_stat"/>
    <addaction name="actionExport_phs"/>
    <addaction name="separator"/>
    <addaction name="actionExit_3"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(247, 255, 174);</string>
   </property>
  </widget>
  <action name="actionSave_Picks_as_CSV">
   <property name="text">
    <string>Save CSV</string>
   </property>
  </action>
  <action name="actionSave_Picks_for_Hypoinverse2000">
   <property name="text">
    <string>Save Hypoinverse2000</string>
   </property>
  </action>
  <action name="actionLoad_JSON">
   <property name="text">
    <string>Open JSON</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAs_JSON">
   <property name="text">
    <string>Save JSON</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionExport_stat">
   <property name="text">
    <string>Export Hyp2k STA file</string>
   </property>
   <property name="statusTip">
    <string>Export Hypoinverse2000 station file in data format #1</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_3">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q, Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_phs">
   <property name="text">
    <string>Export Hyp2k PHS file</string>
   </property>
   <property name="statusTip">
    <string>Export Hypoinverse2000 phase file in format</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stationTree</tabstop>
  <tabstop>qtGraphLayout</tabstop>
  <tabstop>fminSpin</tabstop>
  <tabstop>fmaxSpin</tabstop>
  <tabstop>cornersSpin</tabstop>
  <tabstop>zerophaseCheck</tabstop>
  <tabstop>filterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
